### ISSUE TITLE
Cannot send with Tor on latest master

### ISSUE THREAD

**Issue by quentinlesceller** [mimblewimble/grin-wallet#418]
Thanks to @mably, we found this issue on latest master:

```
grin-wallet --floonet send -d volyuczoexpctup3dzqfybnh4tz4xsh6zow6r3wcd4y7yugb67r3iuyd -f  1
```
fails with
```
Wallet command failed: Generic error: Invalid wallet comm type and destination. method: http, dest: volyuczoexpctup3dzqfybnh4tz4xsh6zow6r3wcd4y7yugb67r3iuyd
```

this same command works on grin v3.1.1.

**Comment by mably** [comment]
Still can't find a way to make a Tor payment with latest master.

If I only put the Tor address it creates a slatepack file.

If I put the full Tor URL starting with http it tells me that http is deprecated.

Using the `--help` cmd line parameter doesn't give any more information about Tor sending.

**Comment by yeastplume** [comment]
@mably This fix was on the 3.1.x branch, so you probably want to try from there. Master is being prepared for 4.0.0 and [slatepack](https://github.com/j01tz/grin-rfcs/blob/slatepack/text/0000-slatepack.md) and isn't quite complete or documented yet.

**Comment by mably** [comment]
Just tried sending using the new slatepack address, it worked like a charm, thanks.
